### ISSUE TITLE
Flux Free Integrationish Testing for Flows

### ISSUE THREAD

**Issue by garside** [Freshly/flow#134]
Integration testing flows with comprehensively tested operations feels like a pain in the ass, but you never really know otherwise right now.

Let's add some way to give you that assurance.

My gut is telling me this is probably gonna manifest as something like a custom matcher for whatever shakes out of #61 

The point is there should be something that fluxes your operations and does a dry run of "could this ever work, regardless of your input".

Ideally the same technology that powers this at runtime will be what powers the "lite integration" test for flow.

I'm at the point where I feel the best way to integration test flow is with a full-scale integration test, like around the controller and everything too. 

tl;dr:

```
That + "could this ever work" assertion + state validations + good operation unit testing = the winning combo for flow test coverage
```

agree/disagree?
